src/panel/uuid.vala

```diff
@@ -28,7 +28,10 @@ namespace LibUUID {
 		} else if ((flags & LibUUID.UUIDFlags.TIME_TYPE) != 0) {
 			LibUUID.generate_time(time);
 		} else if ((flags & LibUUID.UUIDFlags.TIME_SAFE_TYPE) != 0) {
-			LibUUID.generate_time_safe(time);
+#if USE_OPENBSD
```

Comment:
Should move the macro one line up, to be above the "else if" rather than below it. As it stands, if built for OpenBSD and the TIME_SAFE_TYPE flag is passed, nothing will be executed.